src/components/general/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import * as styles from "./styles.less";
+
+type TabsProps = {
+    onTabClick: (tabKey: string) => void,
+    activeTabKey: string,
+    children: any,
+};
+
+const Tabs = ({ onTabClick, activeTabKey, children }: TabsProps) => {
+    const renderTabContent = () => {
```

Comment:
Declaring lots of functions within a functional component can be expensive and lead to memory leaks over time with lots of components and rerenders. Could this be extracted to a seperate component? e.g. `<TabContent activeTabKey={activeTabKey}>{children}</TabContent>`